packages/block-editor/src/components/block-list/style.scss

```diff
@@ -290,6 +290,18 @@
 		clear: both;
 	}
 
+	// Full-wide needs negative margins to accommodate the root container padding.
+	&[data-align="full"],
+	&.alignfull {
+		margin-left: -$block-padding;
```

Comment:
We should retire/rename both of the variables used here, as they are no longer descriptive of their original intent.

Reply:
I kept this style but I moved it to the place where we add the padding to the block wrapper.

Do we really need to add it here? Why?

https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-list/style.scss#L650-L662

Reply:
I don't know... Do you not see the margin issue in master? Is it just a cache ghost ok my machine? Note that I'm using a theme that doesn't provide an editor style.

Reply:
Yes, it seems to happen only for themes without editor styles, still trying to figure out why

Reply:
Actually, I figured it out, it depends on the block. Try Media&Text for instance, you'll notice that it works just fine.

Reply:
it seems that some blocks have an extra "div" around the block (not sure why). so we probably need to remove the ">" for now. (I initially didn't want this style to apply to nested full width blocks) (say a full width image inside a full width group beccause the negative margin would already be on the group)

Reply:
What's a good fix here? I'm unsure how to proceed.

Reply:
Is this fix not only needed for `.alignfull`? We should probably not touch `data-align="full"`.

Reply:
It's for both

Reply:
I need to run some errands, but I'll have a look after that in a bit. I think somehow we need alignfull/wide on the wrapper div.

Reply:
I'm not sure I understand the problem... It seems fines even without editor styles

Reply:
Actually scratch what I said about editor styles. This is also in TwentyTwenty:

<img width="1096" alt="Screenshot 2020-03-28 at 13 46 14" src="https://user-images.githubusercontent.com/1204802/77823319-8a246f80-70fa-11ea-90eb-db365e6d2f7a.png">

It's because of this:

<img width="520" alt="Screenshot 2020-03-28 at 13 45 29" src="https://user-images.githubusercontent.com/1204802/77823325-927caa80-70fa-11ea-90a5-616eee22c43c.png">

That div wraps the Image block, which means it isn't targetted by this:

```
.block-editor-block-list__layout.is-root-container > .block-editor-block-list__block[data-align="full"]
```

I think we need to do two things:

1. Figure out why the image block has an extra div in the editor — was a new fragment introduced?
2. Relax or reconsider the idea of scoping the full-wide rule. 

What do you think, @youknowriad?

Reply:
Yes, I haad the same thoughts here

https://github.com/WordPress/gutenberg/pull/21201#discussion_r399312021

I think we should relax the rule for now and if possible try to find a way to not apply it in inner blocks.

Reply:
I would agree, but your point about the group block was well made. And if we change the current code (https://github.com/WordPress/gutenberg/pull/21201/files#diff-ee2ed3adbe2578628039530c717a9a93R652) to just relax it overall, that regresses the column block with wide blocks inside.